File: benchmark/java/src/main/java/com/benchmark/services/DatabaseService.java
package com.benchmark.services;

import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.Statement;

public class DatabaseService {
    public void getUserById(String id) {
        try {
            Connection conn = DriverManager.getConnection("jdbc:mysql://localhost/test", "root", "password");
            Statement stmt = conn.createStatement();
            // SQL Injection [CWE-89]
            String query = "SELECT * FROM users WHERE id = '" + id + "'";
            stmt.executeQuery(query);
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}

File: benchmark/csharp/Services/DatabaseService.cs
using System.Data.SqlClient;

namespace Benchmark.CSharp.Services
{
    public class DatabaseService
    {
        public void GetUserById(string id)
        {
            using (SqlConnection connection = new SqlConnection("Server=myServerAddress;Database=myDataBase;User Id=myUsername;Password=myPassword;"))
            {
                // SQL Injection [CWE-89]
                string query = "SELECT * FROM Users WHERE Id = '" + id + "'";
                SqlCommand command = new SqlCommand(query, connection);
                
                connection.Open();
                command.ExecuteReader();
            }
        }
    }
}
